flake.nix

```diff
@@ -134,7 +134,7 @@
             # but how should we make it work with other python versions?
             manifold3d = with pkgs.python3Packages; buildPythonPackage {
               pname = "manifold3d";
-              version = "2.2.0";
+              version = "2.2.1";
```

Comment:
Do you think it would be worthwhile to build a little release script to update all the version numbers and ensure they're consistent?

Reply:
sounds good